### DISCUSSION TITLE
How to disable the display of hints when typing?

### DISCUSSION THREAD

**Discussion by ghost** [JanDeDobbeleer/oh-my-posh#3184] [Troubleshoot]
#### I want to disable the display of hints when typing, how can I do this? This is very annoying.

![image](https://user-images.githubusercontent.com/45774876/206322439-a3697538-a341-4a17-a0ae-dace94c7b669.png)

**Comment by JanDeDobbeleer** [comment]
@larxvy this isn't oh-my-posh but PowerShell. See [here](https://devblogs.microsoft.com/powershell/psreadline-2-2-6-enables-predictive-intellisense-by-default/) for more information.

**Reply by ghost** [reply]
@JanDeDobbeleer Thanks a lot, figured it out!
